src/auth/entities/user.entity.ts

```
@@ -97,9 +99,14 @@ export class User {
     example: '2023-01-01T00:00:00.000Z',
   })
   @CreateDateColumn()
-  createdAt: Date;
+  createdAt: Date
+
+  @ApiProperty({
+    description: "Phone number of the user (for SMS notifications)",
```

ESLint

[error] Replace `"Phone·number·of·the·user·(for·SMS·notifications)"` with `'Phone·number·of·the·user·(for·SMS·notifications)'`

(prettier/prettier)